src/AppBundle/Resources/translations/messages.fr.yml

```diff
@@ -484,6 +484,8 @@ form.product_option.add_value.label: Ajouter une valeur
 form.product_option.values.title: Valeurs
 form.product_option_value.value.label: Valeur
 form.product_option_value.price.label: Prix
+form.product_option.additional.label: Cette option est un supplément
```

Comment:
I don't think this text is clear enough.
"Cette option est facultative."

Reply:
The problem is it translates to "this option is optional" 😅